css-three-column-layout/index.html

```diff
@@ -10,10 +10,10 @@
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-  <div class="row">
+  <div class="container">
     <h1 class="text">Designed for Life science. Optimized for SaMD and Digital Health.</h1>
   </div>
-  <div class="row">
+  <div class="container">
```

Comment:
We still want a row for the display: flex, but we want an additional element surrounding it to act as the container.